registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/entity/RegistrationStatusEntity.java

```diff
@@ -94,6 +95,14 @@
 	@Column(name = "reg_process_retry_count")
 	private Integer regProcessRetryCount;
 
+	/** The resume time stamp. */
+	@Column(name = "resumetimestamp")

```

Comment:
can we separate resumetimestamp and below defaultresumeaction to separate with _ as other columns ?

Reply:
I will fix it